meson.build

```diff
@@ -0,0 +1,168 @@
+project('com.github.needleandthread.vocal', 'vala', 'c')
```

Comment:
You probably want to add `version: '2.4.2'` here.

Also consider adding a `meson_version: '>=0.57.0'` or something, to declare a minimum supported version of meson and warn when features from too new a version are used.